### ISSUE TITLE
HCM ICD-10 xref only linked to obstructive HCM

### ISSUE THREAD

**Issue by reality** [monarch-initiative/mondo#6304]
**Mondo term (ID and Label):** hypertrophic cardiomyopathy (MONDO:0005045)

**Xref that should be fixed (ID and label):** ICD-10 xref for hypertrophic cardiomyopathy is currently I42.1, referring specifically to obstructive HCM, should also include I42.2 for other HCM. I usually see associations prefer 'other' where otherwise unspecified, however I'm not sure what the convention is in MONDO, in this case. I see also that there is an ICD-9 xref for 425.4 'other cardiomyopathies', which may be too general, perhaps consider retaining only existing 425.1.

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/) https://orcid.org/0000-0001-9227-0670

**Other comments:** Thank you

**Comment by nicolevasilevsky** [comment]
@reality - thanks for your comments.

- I don't think that I42.1 obstructive HCM is equivalent to this term, the Mondo class is broader (more general) than the source xref, so I updated this accordingly. (MONDO:mondoIsBroaderThanSource)
- I added I42.2 as a Mondo:relatedTo, as this is not equivalent either.
- ICD9:425.1 Cardiomyopathy is more general than the Mondo terms, so I updated this accordingly (MONDO:mondoIsNarrowerThanSource)
